lib/web3-contracts.js

```diff
@@ -1,11 +1,13 @@
 import { useEffect, useRef, useState, useCallback, useMemo } from 'react'
 import { Contract as EthersContract } from 'ethers'
+import environment from 'lib/environment'
 import { getKnownContract } from './known-contracts'
 import { useWeb3Connect } from './web3-connect'
 import { bigNum } from './utils'
 
 // See https://fundraising.aragon.black/components/bonding-curve#pricing-algorithm
 const CONNECTOR_WEIGHT = 250000
+const RINKEBY_CONNECTOR_WEIGHT = 33333
```

Comment:
Maybe we can put this configuration value in a Map, to be similar to `KNOWN_CONTRACTS_BY_ENV`?

Reply:
Let's do it!